books/RayTracingInOneWeekend.html

```diff
@@ -473,31 +473,38 @@
 
 Sending Rays Into the Scene
 ----------------------------
-Now we are ready to turn the corner and make a ray tracer. At the core, the ray tracer sends rays
-through pixels and computes the color seen in the direction of those rays. The involved steps are
-(1) calculate the ray from the eye to the pixel, (2) determine which objects the ray intersects, and
-(3) compute a color for that intersection point. When first developing a ray tracer, I always do a
-simple camera for getting the code up and running. I also make a simple `ray_color(ray)` function
-that returns the color of the background (a simple gradient).
+Now we are ready to turn the corner and make a ray tracer.
+At the core, the ray tracer sends rays through pixels and computes the color seen in the direction
+of those rays.
```

Comment:
Should this be on one line?

Reply:
Yes. It all renders that way, and will be a question that we can revisit when it comes time to prep for release. I'm fine with reflowing everything at that point, but it's a bit nicer to review text changes this way. If I'm wrong on that, I can reflow again at any point.